### PR TITLE
mage fixes

### PR DESCRIPTION
**Pull request by ezrn**
- Fixed static lazer not reducing energy per level
- Fixed inferno spread(forgot to add a 0 last time)
- Made inferno use energy, decreased cooldown back to original
- Fixed blizzard energy
- Fixed glacial blade not consuming energy

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have tested my changes.

### CODE REVIEW

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/sword/StaticLazer.java [hunk 1]
```diff
@@ -141,6 +140,7 @@ public void loadSkillConfig(){
         energyPerSecond = getConfig("energyPerSecond", 24.0, Double.class);
         collisionRadius = getConfig("collisionRadius", 1.8, Double.class);
         explosionRadius = getConfig("explosionRadius", 4.0, Double.class);
+        energyPerSecondReductionPerLevel = getConfig("energyPerSecondReductionPerLevel", 2.0, Double.class);
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/sword/StaticLazer.java:143]: I think adding energyPerSecond and energyPerSecondReductionPerLevel were a mistake here (probably reys fault)

All energy skills have a default configuration

image

Can you change it to use these instead

**Review by Mykindos — CHANGES_REQUESTED** [review]

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/sword/StaticLazer.java (hunk 1 above)
Comment (ezrn) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/sword/StaticLazer.java:143]: energyPerSecondReductionPerLevel is for the energy consumed per second, the skill also has an initial energy cost which is defined by energy and energyDecreasePerLevel

Comment (Mykindos) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/sword/StaticLazer.java:143]: No other skill does that, we typically just use the base energy cost as the activation cost, or pick a relatively low number

Comment (ezrn) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/sword/StaticLazer.java:143]: well both static and now this do. I can change them both to use a higher amount and drain over time

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/sword/Inferno.java [hunk 2]
```diff
@@ -85,14 +86,19 @@ public String[] getDescription(int level) {
...
     public double getFireDuration(int level) {
         return baseFireDuration + level * fireDurationIncreasePerLevel;
     }
 
+    private float getEnergyPerSecond(int level) {
+        return (float) (energy - ((level - 1) * energyDecreasePerLevel));
+    }
+
     public int getNumFlames(int level){
         return baseNumFlames + level * numFlamesIncreasePerLevel;
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/sword/Inferno.java:103]: fix the scaling here to be level -1, same with getFireDuration above (i know you didnt change those in this PR but might as well fix

**Review by Mykindos — CHANGES_REQUESTED** [review]

**Review by Mykindos — APPROVED** [review]

**Pull request merged**